Create LICENSE

BryteLitty opened the pull request:
In this project, I did the following:

- created two sections, header and headline
- added a background image to enhance the look of the website
- used flexbox to place elements in the page
- used linters to check for code errors
- created a new repo on github
- created a new branch after pushing my code
- used the new branch to add a LICENSE.md file
-

BryteLitty committed:
Create LICENSE

Abenezer-Tilahun reviewed (changes requested):
Hi @BryteLitty    , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages


## Required Changes ♻️

- [ ]  For Making Your HTML code semantically correct I recommend you to use a `section` tag instead of the `div` tag . Because It represents a generic standalone section of the document, so Please use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
https://github.com/BryteLitty/portfolio-project/blob/76da5445dcf020a800d8340a9508e23b3003e6e2/index.html#L28-L68

- [ ] Good job for using a `ul` tag to group them, and `li` per icon; each icon with a link (`a` tag). I recommend you to use the `a` tag to let you connect your social networks with the icons, using `href.`
https://github.com/BryteLitty/portfolio-project/blob/76da5445dcf020a800d8340a9508e23b3003e6e2/index.html#L43-L64

- [ ] You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly adjust the section to the margins (top, bottom, left, right) defined in the design and for social media icons try to use an icon just like the design.
Your Project Look Likes this  : 

![c](https://user-images.githubusercontent.com/93598189/191781427-45ab99ab-40ec-4b2a-a78d-36498e7a5ad1.PNG)

The Figma design Look Likes this : 

![c1](https://user-images.githubusercontent.com/93598189/191781590-4adeaf95-7fe8-4ec6-ae0d-e2faf3c58e98.PNG)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BryteLitty committed:
udate code after review

BryteLitty committed:
fix linter error

BryteLitty committed:
fix linter error

BryteLitty commented:
Hi @Abenezer-Tilahun 
As recommended, I have made the following changes to my code.

- added semantic tag instead of div
- adjusted spacing between elements
- added my handles to the social links using href

youmari reviewed (approved):
# Project Approved :trophy: :tada: 🟢 

Hi @BryteLitty 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
